app/controllers/test/components/student/track_list_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Test::Components::Student::TrackListController < ApplicationController
+  def show
```

Comment:
I've started use of our serializers and search commands in our test controller.

Reply:
Great!

Reply:
Nice! 🙂